metricbeat/docs/fields.asciidoc

```diff
@@ -32795,6 +32795,369 @@ type: long
 
 --
 
+[float]
+=== crawler
+
+Aggregate stats on the functioning of the crawler subsystem within Enterprise Search.
+
+
+[float]
+=== global
+
+Global deployment-wide metrics for the crawler.
+
+
+[float]
+=== crawl_requests
+
+Crawl request summary for the deployment.
+
+
+*`enterprisesearch.stats.crawler.global.crawl_requests.pending`*::
++
+--
+Total number of crawl requests waiting to be processed.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.global.crawl_requests.active`*::
++
+--
+Total number of crawl requests cyrrently being processed (running crawls).
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.global.crawl_requests.successful`*::
++
+--
+Total number of crawl requests that have succeeded.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.global.crawl_requests.failed`*::
++
+--
+Total number of failed crawl requests.
+
+type: long
+
+--
+
+[float]
+=== node
+
+Node-level statistics for the crawler.
+
+
+*`enterprisesearch.stats.crawler.node.pages_visited`*::
++
+--
+Total number of pages visited by the crawler since the instance start.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.node.urls_allowed`*::
++
+--
+Total number of URLs allowed by the crawler during discovery since the instance start.
+
+type: long
+
+--
+
+[float]
+=== urls_denied
+
+Total number of URLs debied by the crawler during discovery since the instance start, broken down by deny reason.
+
+
+*`enterprisesearch.stats.crawler.node.urls_denied.already_seen`*::
++
+--
+Total number of URLs not followed becuase of URL de-duplication (each URL is visited only once).
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.node.urls_denied.domain_filter_denied`*::
++
+--
+Total number of URLs denied because of an unknown domain.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.node.urls_denied.incorrect_protocol`*::
++
+--
+Total number of URLs with incorrect/invalid/unsupported protocols.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.node.urls_denied.link_too_deep`*::
++
+--
+Total number of URLs not followed due to crawl depth limits.
+
+type: long
+
+--
+
+*`enterprisesearch.stats.crawler.node.urls_denied.nofollow`*::
++
+--
+Total number of URLs denied due to a nofollow meta tag or URL attribute.
```

Comment:
What's a URL attribute in this context?

Reply:
Doh, should have been a link attribute there. Fixing now.